### ISSUE TITLE
license and/or pip release?

### ISSUE THREAD

**Issue by davedgd** [clemtoy/WNAffect#7]
@clemtoy: thanks for putting together this code and providing it on GitHub based on the [Stack Overflow post](https://stackoverflow.com/questions/27943396/using-wn-affect-to-detect-emotion-mood-of-a-string/31032077#31032077) that came up a few years back.

Regarding the code, would it be possible to add a license to it to facilitate including it in other projects? You have included license information on other projects such as [pptree](https://github.com/clemtoy/pptree/blob/master/LICENSE.txt), so I'm hoping this would be a simple addition.

On a related point, I was wondering if you've considered making the WNAffect code available via pip to make its use even easier. If it's an issue of time, I'd be happy to help out with this (assuming you would be interested), since I've been making very minor improvements to this code over the years and adding compatibility for newer versions of Python (e.g., in Python 3.9.2+ I noticed some of the `if` statements in `printTree` produce minor warnings), path issues, etc. Alternatively, I'd be happy to submit these as pull requests as well.

TL;DR: this code has been extremely helpful over the years, and I'm hoping there's a way to make it more easily accessible so that other projects can benefit from it!
